plugins/inputs/kernel/kernel.go

```diff
@@ -13,6 +13,7 @@ import (
 
 	"github.com/influxdata/telegraf"
 	"github.com/influxdata/telegraf/plugins/inputs"
+	"github.com/prometheus/procfs"
```

Comment:
We usually have 3 groups of imports, builtins, 3rd-party libs and telegraf internal ones (in this order), each ordered alphabetically within the group. This make spotting external dependencies easier... So please do
```suggestion
	"github.com/prometheus/procfs"
	
	"github.com/influxdata/telegraf"
	"github.com/influxdata/telegraf/plugins/inputs"
```
Same for the `psi.go` and test files.

Reply:
I use `goimports` as a replacement for `gofmt`, which only arranges imports into stdlib and non-stdlib. I also have automatic `goimports` on save configured for all my editors. Not sure how to easily get over this one.

Reply:
Looks like there's a `-local` flag that needs some configuration (it never had an "automatically detect from `go.mod`" feature). I ran it on `*.go` files.